templates/variable_inplace.cpp.in

```diff
@@ -18,7 +18,7 @@ Variable &@NAME@(Variable &lhs, const VariableConstView &rhs) {
 }
 
 VariableView @NAME@(const VariableView &lhs, const VariableConstView &rhs) {
-  transform_in_place(lhs, rhs, core::element::@OPNAME@, std::string_view("OPNAME"));
+  transform_in_place(lhs, rhs, core::element::@OPNAME@, std::string_view("@OPNAME@"));
```

Comment:
What kind of problems was this causing?
Was this not working on some platforms?

Reply:
When passing unsupported types to the operation, it printed "Cannot apply OPNAME to dtypes ...". Now it prints the actual name of the op